tests/coalaTest.py

```diff
@@ -326,13 +326,14 @@ def test_coala_without_color(self):
 
     def test_coala_ignore_file(self):
         with bear_test_module(), \
-                prepare_file(['#fixme  '], None) as (lines, filename):
+                prepare_file(['#fixme'], None) as (lines, filename):
             retval, stdout, stderr = execute_coala(
                     coala.main, 'coala',
+                    '-c', os.devnull,
                     '--non-interactive',
                     '-f', re.escape(filename),
                     '-i', re.escape(filename),
-                    '-b', 'ErrorTestBear')
-            self.assertIn(
-                    "No files matching '{}' were found.".format(filename),
-                    stderr)
+                    '-b', 'LineCountTestBear')
+            self.assertNotIn(stdout, 'This file has''lines')
```

Comment:
Is the second argument right? It just amounts to `'This file haslines'`

Reply:
should be `this file has 1 lines`

Reply:
@Mixih then howcome the tests passed?

Reply:
Idk how it passes, I'll have to take a look.

Reply:
Oh nevermind, if the ignore flag doesn't work, the bear prints `this file has 2 lines`, since the assert is looking for `this file haslines`, it still passes. Sorry about that.